pkg/maven/maven.go

```diff
@@ -92,14 +91,21 @@ func Execute(options *ExecuteOptions, command mavenExecRunner) (string, error) {
 // evaluate a given expression from a pom file. This allows to retrieve the value of - for
 // example - 'project.version' from a pom file exactly as Maven itself evaluates it.
 func Evaluate(options *EvaluateOptions, expression string, command mavenExecRunner) (string, error) {
+	return EvaluateWithCustomDefines(options, expression, command, []string{})
+}
+
+// EvaluateWithCustomDefines is the same as Evaluate, but allows to pass in additionalDefines
+func EvaluateWithCustomDefines(options *EvaluateOptions, expression string, command mavenExecRunner, additionalDefines []string) (string, error) {
 	expressionDefine := "-Dexpression=" + expression
+	defines := []string{expressionDefine, "-DforceStdout", "-q"}
+	defines = append(defines, additionalDefines...)
 	executeOptions := ExecuteOptions{
 		PomPath:             options.PomPath,
 		M2Path:              options.M2Path,
 		ProjectSettingsFile: options.ProjectSettingsFile,
 		GlobalSettingsFile:  options.GlobalSettingsFile,
 		Goals:               []string{"org.apache.maven.plugins:maven-help-plugin:3.1.0:evaluate"},
-		Defines:             []string{expressionDefine, "-DforceStdout", "-q"},
+		Defines:             defines,
```

Comment:
I would like the solution better to extend the `EvaluateOptions` struct with an optional field `Defines []string`. Having a struct passed into `Evaluate()` makes such an extension easily possible without introducing new methods.